Entity/DeliveryStatus.php

```diff
@@ -9,6 +9,10 @@
  */
 class DeliveryStatus
 {
+
+    const AVAILABLE = 1;
+    const NOT_AVAILABLE = 2;
```

Comment:
hmm. why use integers? it makes debugging and inspecting the database a real pain. I know we do this everywhere else, but just saying. /cc @chirimoya @danrot @wachterjohannes